### ISSUE TITLE
Accessibility - Zooming at 400% removes the sidebar

### ISSUE THREAD

**Issue by sebastienlevert** [storybookjs/storybook#17289]
**Describe the bug**
When zooming at 400% using the browser native capabilities, Storybook simply removes the sidebar and it becomes impossible to go back to it to change story.

 **Test Environment:**   
-  Browser: Microsoft edge Version 89.0.63 (Official Build) (64-bit) 
-  URL: https://mgt.dev/next
-  OS Version: Windows 10 Version 20H2 (OS Build 19042.685) 

 **Repro Steps:** 

1. Open the URL:  https://mgt.dev/next in edge browser.                                                                                                                                                      
2. Navigate to browser settings and change the resolution into 400%.                                                                                           
3. check whether the page content is properly visible or not when the page is zoomed to 400%. 

**Actual Result:** 
Page content is not properly visible when the page is zoomed to 400%

*Zooming at 300%*
![image](https://user-images.githubusercontent.com/7620955/150148714-773d657c-2749-4e7f-b957-a439a999d3fc.png)

*Zooming at 400%*
![image](https://user-images.githubusercontent.com/7620955/150148809-741b10fc-6c58-4e48-907b-5f986ee2ee92.png)

**Expected Result:**  
page content should be properly visible without loss of any information when the page is zoomed to 400%. 

 **User Impact:** 
 People with low vision will face difficulty if the contents or the functionality is not same as in the normal mode so the end users won’t be able to use the full functionality of the UI so the end users will get confused& this may lead to disorientation due to this issue.  

We also have a tracked issue on our repo : https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/1094

**Comment by pocka** [comment]
@shilman I believe we should close this. I can close this but I'm not sure which label is appropriate for this kind of case...

This was due to an element (footer) added by the user hiding Storybook's footer.
https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/1613#issuecomment-1070709953

> ... still have the same behavior (footer hiding the sidebar / canvas selector)...

The linked website now displays Storybook's footer correctly. This below screenshot is of Firefox (macOS) on a WQHD display scaled to 1920x1080, with 400% browser zoom.

<img width="961" alt="Screenshot 2024-09-13 at 21 58 24" src="https://github.com/user-attachments/assets/c08a5795-2766-46f7-b5ee-baa8a3fec6a2">
